server/command/command.go

```diff
@@ -153,17 +153,23 @@ type command struct {
 	handler HandlerMap
 }
 
-func (c *command) help(_ ...string) (*model.CommandResponse, error) {
+func (c *command) help(userID string, _ ...string) (*model.CommandResponse, error) {
 	helpText := helpTextHeader + helpText
-	return c.postCommandResponse(helpText), nil
+	return c.postCommandResponse(userID, helpText), nil
 }
 
-func (c *command) postCommandResponse(text string) *model.CommandResponse {
-	return &model.CommandResponse{Text: text}
+func (c *command) postCommandResponse(userID, text string) *model.CommandResponse {
+	post := &model.Post{
+		UserId:    c.splunk.BotUser(),
+		ChannelId: c.args.ChannelId,
+		Message:   text,
+	}
+	c.splunk.SendEphemeralPost(userID, post)
+	return &model.CommandResponse{}
 }
 
 func (c *command) responsef(format string, args ...interface{}) *model.CommandResponse {
-	return c.postCommandResponse(fmt.Sprintf(format, args...))
+	return c.postCommandResponse("", fmt.Sprintf(format, args...))
```

Comment:
@mickmister can we remove this function , seems not being used in this repo
is it used by some other plugin to interact

can you let me know on this

Reply:
@sibasankarnayak Yes if code is not being used (dead code) it can and should be removed. No need to ask :) but if you think it's important to note just mention that you removed it in the PR. Thanks for finding this Siba!

Reply:
@mickmister sure